crates/wasi-http/src/struct.rs

```diff
@@ -66,6 +72,15 @@ impl ActiveResponse {
     }
 }
 
+impl Stream {
+    pub fn new() -> Self {
+        Self {
```

Comment:
If we had a `Default` trait impl, this could have just called `Self::default()`

Reply:
done.